src/components/profile/profile.js

```diff
@@ -187,10 +190,23 @@ export class Profile extends React.Component {
                 {this.state.data.handle ? "@" : null}
                 {this.state.data.handle}
               </Card.Meta>
-              <Card.Description textAlign="center"> {data.description}</Card.Description>
+              <Card.Description style={{ marginTop: "1em" }}>
+                {data.branch}
+              </Card.Description>
+              <Card.Description style={{ marginBottom: "1em" }}>
+                {data.degreeSem}
+              </Card.Description>
+              <p styleName="style.gray">{data.description}</p>
             </div>
-            <div className="center aligned content" style={{ border: "0", textAlign: "center" }}>
-              <LinkDisplay handle={this.props.handle} theme={this.props.theme} />
+            <div
+              className="center aligned content"
```

Comment:
I am assuming center aligned content are semantic classes. Why have you used them like this? We added the package semanticui-react to avoid such things.
Anyway, whatever you are trying to do can be achieved using flexbox.

Reply:
I didn't make these changes, it was already there. And since you mentioned not to refactor the previous code, I didn't touch this part, yet. :neutral_face:

Reply:
Oh, okay. I said for the formatting changes though. But anyway, we can do all of this in a separate PR.